Frontend/ui-customer/src/Components/BusTicketInfoCard/View.purs

```diff
@@ -0,0 +1,262 @@
+{-
+
+  Copyright 2022-23, Juspay India Pvt Ltd
+
+  This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+  as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+  is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Components.BusTicketInfoCard.View where
+
+import Common.Types.App
+import Animation (fadeIn, fadeInWithDelay)
+import Common.Types.App (LazyCheck(..))
+import Components.BusTicketInfoCard.Controller
+import Components.PrimaryButton as PrimaryButton
+import Components.SourceToDestination as SourceToDestination
+import Data.Array as Array
+import Data.Maybe (fromMaybe)
+import Data.String (Pattern(..), split, length, take, drop, replaceAll, Replacement(..), contains, toLower)
+import Data.String.CodeUnits (fromCharArray, toCharArray)
+import Debug (spy)
+import Effect (Effect)
+import Effect.Aff (launchAff_)
```

Comment:
Please remove all the unnecessary import statements. I can see a lot of unused imports here